pylint/extensions/_check_docs_utils.py

```diff
@@ -33,6 +34,67 @@ def space_indentation(s):
     return len(s) - len(s.lstrip(' '))
 
 
+def is_property(name, node):
+    """Check if the node is a property with the given name.
+
+    :param name: The name of the property to look for.
+    :type name: str
+    :param node: The node to check.
+    :type node: astroid.Node
+
+    :rtype: bool
+    :returns: Whether the given node is a property with the given name (True)
+        or not (False).
+    """
+    return (
+        isinstance(node, astroid.FunctionDef) and
+        node.name == name and
+        any(name.endswith("property") for name in node.decoratornames())
```

Comment:
Even better, there's `utils.decorated_with_property`! I knew this must have existed somewhere ;)